src/Generation/Generator/Renderer/Internal/Error.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Generator.Model;
+
+namespace Generator.Renderer.Internal;
+
+internal static class Error
+{
+    public static string Render(GirModel.Callable callable)
+    {
+        if (!callable.Throws)
+            return string.Empty;
+
+        // Add a separator if there are any previous parameters.
+        var separator = string.Empty;
+        if (callable.Parameters.Any() || callable.InstanceParameter is not null)
+            separator = ",";
```

Comment:
The separator probably should contain the whitespace to avoid leaking a space in case there is no other parameter?